### DISCUSSION TITLE
Setting allowed minimum and maximum values within editable option byte array?

### DISCUSSION THREAD

**Discussion by MiqViq** [Spirik/GEM#92] [General]
Is there a way to set custom value ranges for allowed minimum and maximum values within editable items like byte, int, float, double?

byte myByte = 16;
GEMItem menuItemByte("Byte:", myByte [, ALLOWED_MIN_VAL, ALLOWED_MAX_VAL] );

**Comment by Spirik** [comment]
This can be achieved through [Validation callback](https://github.com/Spirik/GEM/wiki/How-to#add-menu-item-callback-basic), which will be fired when menu item is saved, and in which values can be overridden if they fall out of range.
